tests/Doctrine/Tests/DBAL/Functional/Ticket/DBAL4283Test.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Functional\Ticket;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver;
+use Doctrine\DBAL\Platforms\SQLServer2012Platform;
+use Doctrine\DBAL\Schema\Column;
+use Doctrine\DBAL\Schema\ColumnDiff;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\SQLServerSchemaManager;
+use Doctrine\DBAL\Schema\TableDiff;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+use function sprintf;
+
+class DBAL4283Test extends DbalFunctionalTestCase
```

Comment:
If that was really a functional test it would not only work for some jobs of the build matrix, and there would be something to skip it in other jobs. IMO, the test should demonstrate that the bug described at #4283 , which occurs when executing an SQL query does not happen anymore. This means you will have to execute that query at some point.

Reply:
yes - this is test that I merged from another PR

The issue is that DBAL accepts unquoted/quoted names and that is not tested extensively enought across databases.

I now added cross-vendor functional test. I also verified, that the original issue is fixed by reverting the fix.

One last issue - tests with OCI are failing - can you please advise how to fix?